RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/SupportedClusters.java

```diff
@@ -24,6 +24,13 @@ private static List<ContainerVersion> sources() {
         );
     }
 
+    public static List<ContainerVersion> extendedSources() {
```

Comment:
Could you add overview comment for why some sources should be on this list versus the one above?